reefperf/director.py

```diff
@@ -1,13 +1,18 @@
 import itertools
+import logging
 
+from logfury.v0_1 import DefaultTraceMeta
```

Comment:
you should not use DefaultTraceMeta directly, but pull a dependency into one file and then use that in the rest of the files, like this: https://github.com/Backblaze/B2_Command_Line_Tool/blob/master/b2/utils.py#L315

logfury should probably document it...

This is because if one day you will want to upgrade to logfury.v0_2 interface or customize it in any way, you will want to modify one file and not every file in your project